docker/conda/environments/cuda11.8_dev.yml

```diff
@@ -65,6 +65,7 @@ dependencies:
     - jupyterlab
     - libgrpc>=1.49
     - librdkafka=1.9.2
+    - libwebp>=1.3.2 # Required for CVE mitigation: https://nvd.nist.gov/vuln/detail/CVE-2023-4863
     - mlflow>=2.2.1,<3
```

Comment:
Update to <2.7 ?

Reply:
Ok, then the examples.yaml needs to be updated to 3.0, there are conflicting versions across files.